packages/theming/atlas/src/themesource/atlas_core/web/core/base/mixins/_animations.scss

```diff
@@ -10,37 +10,41 @@
     $delay: 0s,
     $duration: 0.3s,
     $property: all,
-    $timing-fucntion: cubic-bezier(0.4, 0, 0.2, 1)
+    $timing-function: cubic-bezier(0.4, 0, 0.2, 1)
 ) {
-    transition: $property $duration $delay $timing-fucntion;
-    transform-style: $style;
+    @if not $exclude-animations {
```

Comment:
Is this checked here due to the fact that you cannot wrap a mixin inside a mixin?

Reply:
Nope, just to prevent other widgets requiring this mixin to throw an error saying it was not imported :)